documentation/src/main/asciidoc/topics/protobuf/Language.proto

```diff
@@ -0,0 +1,13 @@
+enum Language {
+
+   EN = 0;
+
+   DE = 1;
+
+   IT = 2;
+
+   E = 3;
```

Comment:
What's E? ES ?

Reply:
changed

Reply:
The example match the Language.java class, so it should not be changed as I've used this to demonstrate the use of the attributes of EnumValue!

Reply:
@oraNod ^

Reply:
This is how it works currently

Reply:
@wfink ack. I will change both java and proto back.